utils/build.ps1

```diff
@@ -342,11 +342,41 @@ $KnownNDKs = @{
   }
 }
 
-
 $IsCrossCompiling = $HostArchName -ne $BuildArchName
 
 $TimingData = New-Object System.Collections.Generic.List[System.Object]
 
+function Add-TimingData {
+  param
+  (
+    [Parameter(Mandatory)]
+    [string] $Arch,
+    [Parameter(Mandatory)]
+    [string] $Platform,
```

Comment:
Probably a good idea to keep this as a `Platform` type and use `$Platform.ToString()`.